src/app/container/tool-file-editor/tool-file-editor.component.ts

```diff
@@ -66,17 +71,20 @@ export class ToolFileEditorComponent extends FileEditing {
    * Splits up the sourcefiles for the version into descriptor files and test parameter files
    */
   loadVersionSourcefiles(): void {
-    this.descriptorFiles = JSON.parse(JSON.stringify(this.getDescriptorFiles(this.currentVersion.sourceFiles)));
-    this.testParameterFiles = JSON.parse(JSON.stringify(this.getTestFiles(this.currentVersion.sourceFiles)));
-    this.dockerFile = JSON.parse(JSON.stringify(this.getDockerFile(this.currentVersion.sourceFiles)));
+    this.containerTagsService.getTagsSourcefiles(this.id, this.currentVersion.id).subscribe((sourcefiles: Array<SourceFile>) => {
+      this.originalSourceFiles = sourcefiles;
+      this.descriptorFiles = JSON.parse(JSON.stringify(this.getDescriptorFiles(this.originalSourceFiles)));
+      this.testParameterFiles = JSON.parse(JSON.stringify(this.getTestFiles(this.originalSourceFiles)));
+      this.dockerFile = JSON.parse(JSON.stringify(this.getDockerFile(this.originalSourceFiles)));
+    });
   }
 
   /**
    * Combines sourcefiles into one array
    * @return {Array<SourceFile>} Array of sourcefiles
    */
-  getCombinedSourceFiles(): Array<SourceFile> {
-    let baseFiles: Array<SourceFile> = [];
+  getCombinedSourceFiles(): Array<any> {
```

Comment:
We're trying to move away from `any` - seems like this could still be `Array<SourceFile>` (looks like this.descriptorFiles already has that type).

Reply:
changed to `Array<SourceFile>`